jnigen/lib/src/tools/maven_tools.dart

```diff
@@ -126,17 +153,29 @@ ${depDecls.join("\n")}
   }
 }
 
-/// Maven dependency with group ID, artifact ID, and version.
 class MavenDependency {
   MavenDependency(this.groupID, this.artifactID, this.version,
       {this.otherTags = const {}});
   factory MavenDependency.fromString(String fullName) {
     final components = fullName.split(':');
-    if (components.length != 3) {
+    if (components.length < 3) {
```

Comment:
What's the idea behind this change?

I think If we don't handle all the optional appends in maven coordinate syntax, rather not allow them in the first place.